docs/public/presets/toml/nerd-font-symbols.toml

```diff
@@ -47,7 +47,7 @@ symbol = " "
 symbol = " "
 
 [gcloud]
-symbol = "  "
+symbol = " "
```

Comment:
I looked at this, and some terminals actually render this one as double-width (requiring two spaces), let's replace this with `\ue7f1` (`nf-dev-googlecloud`) which appears to work better in this matter and is specific to gcloud.